connect_dataplane_to_lightup.sh

```diff
@@ -34,16 +33,16 @@ if [[ ${reverse_port_list-} ]]; then
     echo "#!/usr/bin/env bash" > generated/lightup-dataplane-connect-service.sh
     echo  \
     "AUTOSSH_DEBUG=1 \
-    AUTOSSH_LOGFILE=${this_dir}/lightup-dataplane-connect.service.log \
-    AUTOSSH_PIDFILE=${this_dir}/lightup-dataplane-connect.service.pid \
+    AUTOSSH_LOGFILE=/opt/lightup/lightup-dataplane-connect.service.log \
+    AUTOSSH_PIDFILE=/opt/lightup/lightup-dataplane-connect.service.pid \
```

Comment:
Will we have permissions to write it here?

Reply:
yup, this has been tested and works